tensorflow/core/kernels/BUILD

```diff
@@ -2413,6 +2412,7 @@ cc_library(
     hdrs = ["range_sampler.h"],
     visibility = ["//visibility:private"],
     deps = [
+        "//tensorflow/core:framework",
```

Comment:
We can also remove the dependency here since `random_distributions.h` doesn't depend on it anymore.

Reply:
Please also change this to `numeric_types`.
```suggestion
        "//tensorflow/core/framework:numeric_types",
```